mlmodel/docs/readme_session.py

```diff
@@ -1 +1,320 @@
-../../docs/readme_session.py
+import json
```

Comment:
I'm very confused by this file. What is this file for?

Reply:
See comment above.